addons/frag/XEH_postInit.sqf

```diff
@@ -19,10 +19,9 @@ if(isServer) then {
     ["ace_firedPlayerVehicle", DFUNC(fired)] call CBA_fnc_addEventHandler;
     ["ace_firedNonPlayerVehicle", DFUNC(fired)] call CBA_fnc_addEventHandler;
 
-    [FUNC(masterPFH), 0, []] call CBA_fnc_addPerFrameHandler;
-
+    [FUNC(masterPFH)] call CBA_fnc_addPerFrameHandler;
```

Comment:
Leave this as `[FUNC(masterPFH), 0, []] call CBA_fnc_addPerFrameHandler;`
`ENABLE_PERFORMANCE_COUNTERS` has a macro that expects 3 args.